apps/src/templates/teacherDashboard/RosterDialog.jsx

```diff
@@ -163,11 +164,51 @@ class RosterDialog extends React.Component {
       classrooms.find(classroom => {
         return classroom.id === this.state.selectedId;
       }).name;
-
     this.props.handleImport(this.state.selectedId, selectedName);
     this.setState({selectedId: null});
   };
 
+  // create new function for redirect to NewEditPage
+  redirectToEditSectionPage = sectionId => {
+    const redirectUrl = '/sections/' + sectionId + '/edit';
+    window.location.href = redirectUrl;
+  };
+
+  // Creates the section and redirects to the edit page
+  handleRedirect = () => {
```

Comment:
Note that much of this code was taken from `teacherSectionsRedux`, specifically the `importOrUpdateRoster` [function. ](https://github.com/code-dot-org/code-dot-org/blob/d075fe3534f672ab4204ecf20a16d78a673af6d2/apps/src/templates/teacherDashboard/teacherSectionsRedux.js#L545)